Tests/L0/Common-SonarQubeHelpers/InvokeRestApi.ps1

```diff
@@ -0,0 +1,69 @@
+[CmdletBinding()]
+param()
+
+. $PSScriptRoot\..\..\lib\Initialize-Test.ps1
+. $PSScriptRoot\..\..\..\Tasks\SonarQubePostTest\Common\SonarQubeHelpers\SonarQubeHelper.ps1
+
+# Common setup
+$distributedTaskContext = 'Some distributed task context'
+Register-Mock Get-TaskVariable { 'http://testsqserver:9999' } -- -Context $distributedTaskContext -Name 'MSBuild.SonarQube.HostUrl'
```

Comment:
Register-Mock and Assert-WasCalled have been added by the build team to help testing.